docs/guide/snaps-development-guide.md

```diff
@@ -376,6 +380,14 @@ Follow these instructions to inspect the background process and view its console
   - Click on "Details"
   - Under "Inspect Views", click `background.html`
 
+### Reinstalling your snap
+Every time you make a change to the source code of your snap, or modify the snap bundle manually, you'll have to reinstall your snap. The currently recommended way to do this is:
+
+1. Go to Metamask > Settings > Snaps
```

Comment:
When you are building your snap and running it locally you don't need to uninstall the snap to reinstall it. Instead you can just call `wallet_enable` and it should re-install the snap with the latest bundle.

Reply:
@FrederikBolding that's good to know! I didn't see that part in the instructions. It's weird because there's no feedback when you call `wallet_enable` on an already installed snap. Thanks!